WooCommerce/Classes/Extensions/UIColor+Helpers.swift

```diff
@@ -1,6 +1,19 @@
 import UIKit
 
 extension UIColor {
+
+    /// Creates a new `UIImage` using the current `UIColor` as the fill
+    ///
+    /// - Parameter size: Size of the image returned — defaults to 1x1
+    /// - Returns: UIImage instance
+    ///
+    func image(_ size: CGSize = CGSize(width: 1, height: 1)) -> UIImage {
```

Comment:
Nice, I would have probably done it the other way around (an extension on UIImage providing an initialiser with a color) but I like how this would read at the call site.